### PR TITLE
Fix timeline section

### PR DESCRIPTION
**Pull request by demiwu96**
Trello card: https://trello.com/c/pVhEejgd

### CODE REVIEW

**Comment by BBoutet1** [comment]
@demiwu96  there is a conflict in AboutUs/TimelineSection.js

**Comment by demiwu96** [comment]
> @demiwu96 there is a conflict in AboutUs/TimelineSection.js

fixed

**Review by gh0stl0nely — DISMISSED** [review]
Nicely done

File: src/components/AboutUs/TimelineSection.js [hunk 1]
```diff
@@ -8,254 +8,306 @@ import Pin from '../../images/about-us/timeline/pin.svg';
...
-        display: 'none',
-        [theme.breakpoints.down('sm')]: {
-            display: 'block',
-            marginRight: 'calc(100% - 82px)'
-        },
+  },
+  tealBackground: {
+    background: 'linear-gradient(180deg, #F5FDFD 0%, #FFFFFF 50%)',
+    padding: '50px 20px 0px 0px',
+
```
Comment (hanlinc27) [inline src/components/AboutUs/TimelineSection.js]: delete empty line space

File: src/components/AboutUs/TimelineSection.js [hunk 2]
```diff
@@ -8,254 +8,306 @@ import Pin from '../../images/about-us/timeline/pin.svg';
...
-        // Tablet Above to Laptop/ Desktop
-        [theme.breakpoints.between('sm', 'md')]: {
-            fontSize: '25px',
-        },
-        // Tablet below to Mobile
-        [theme.breakpoints.down('sm')]: {
-            fontSize: '14px',
-        },
+  },
+  yearCol: {
```
Comment (hanlinc27) [inline src/components/AboutUs/TimelineSection.js:42]: align with rest of text (shift right)

File: src/components/AboutUs/TimelineSection.js [hunk 3]
```diff
@@ -8,254 +8,306 @@ import Pin from '../../images/about-us/timeline/pin.svg';
...
+    },
+  },
+  bg2: {
+    zIndex: 1,
+    position: 'absolute',
+    // float:'right',
+    right: '0',
+    height: '209px',
+    top: '20px',
+    // width: '1050%',
```
Comment (hanlinc27) [inline src/components/AboutUs/TimelineSection.js]: delete

File: src/components/AboutUs/TimelineSection.js [hunk 4]
```diff
@@ -8,254 +8,306 @@ import Pin from '../../images/about-us/timeline/pin.svg';
...
+    },
+    // Tablet below to Mobile
+    [theme.breakpoints.down('sm')]: {
+      width: '30%',
+    },
+  },
+  bg2: {
+    zIndex: 1,
+    position: 'absolute',
+    // float:'right',
```
Comment (hanlinc27) [inline src/components/AboutUs/TimelineSection.js]: delete

**Review by hanlinc27 — CHANGES_REQUESTED** [review]
looks really good! just some tiny formatting fixes

File: src/components/AboutUs/TimelineSection.js (hunk 2 above)
Comment (demiwu96) [reply src/components/AboutUs/TimelineSection.js:42]: The red parts are deleted, so everything in green should align with each other

**Review by hanlinc27 — APPROVED** [review]

**Review by airshiprook — APPROVED** [review]

**Pull request merged**